ModernUO.Serialization.SchemaGenerator/SourceCodeAnalysis.cs

```diff
@@ -34,13 +36,12 @@ public static ParallelQuery<Project> GetProjects(string solutionPath)
         MSBuildLocator.RegisterDefaults();
```

Comment:
With Microsoft.CodeAnalysis.Workspace.MSBuild 4.9+ project loading occurs in a separate process. So it is no longer necessary to locate and register MSBuild unless you are using MSBuild types directly in your program.

Reply:
Thanks!